src/storm/storage/SparseMatrix.h

```diff
@@ -659,24 +659,30 @@ class SparseMatrix {
      * This function makes the given rows absorbing.
      *
      * @param rows A bit vector indicating which rows are to be made absorbing.
+     * @param dropZeroEntries if true, zero entries resulting from the transformation are dropped from the matrix after the transformation.
+     * This action may impact the performance significantly.
```

Comment:
This is a bit vague, maybe something like
```suggestion
     * Dropping zero entries takes time linear in the number of matrix entries.
```